react/features/toolbox/constants.ts

```diff
@@ -34,7 +34,43 @@ export const THRESHOLDS = [
     }
 ];
 
-export const NOT_APPLICABLE = 'N/A';
+/**
+ * Main toolbar buttons priority used to determine which button should be picked to fill empty spaces for disabled
```

Comment:
Style, weird comment syntax?

Reply:
what do you mean?

The comment is:
```
/**
 *
 */
```

Or is the text the problem?

Reply:
nvm, it's the text wrapping it's ok!